app/airq/models/clients.py

```diff
@@ -301,3 +305,22 @@ def maybe_notify(self) -> bool:
 
     def log_event(self, event_type: EventType, **event_data: typing.Any) -> Event:
         return Event.query.create(self.id, event_type, **event_data)
+
+    def get_last_client_event(self, exclude_type: EventType) -> Event:
+        return (
+            Event.query.filter(Event.client_id == self.id)
+            .filter(Event.type_code != exclude_type)
+            .order_by(Event.timestamp.desc())
+            .first()
+        )
+
+    def should_accept_feedback(self) -> bool:
+        last_event = self.get_last_client_event(
+            exclude_type=EventType.ALERT  # exclude alerts
```

Comment:
Instead of this, you could make `get_last_client_event` exclude alerts by default. Or to be more explicit, you could rename `get_last_client_event` to `get_last_inbound_event`, since alerts are outbound. And then have it exclude alerts by default.

Reply:
I'll go with `get_last_inbound_event` and exclude alerts by default